package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@rezact/create-app",
-  "version": "1.0.9",
+  "version": "1.5.0",
```

Comment:
Excellent Work! Can you adjust this to 1.1.0 ?

I believe these changes merits a single minor bump in versioning, unless you have a compelling argument for the larger jump?